radius.c

```diff
@@ -61,39 +61,199 @@ ZEND_DECLARE_MODULE_GLOBALS(radius)
 /* True global resources - no need for thread safety here */
 static int le_radius;
 
+/* {{{ arginfo */
+ZEND_BEGIN_ARG_INFO_EX(arginfo_radius_auth_open, 0, 0, 0)
```

Comment:
I'd suggest that you should use gen_stub.php (in the build directory) for defining arginfo structures. I recently created a similar PR for the scrypt extension: https://github.com/DomBlack/php-scrypt/pull/63

Reply:
Yes. I am already there and nearly complete with a rebase against the latest (8.1) skeleton.